test/test_prompt_generators.py

```diff
@@ -17,18 +19,22 @@ def test_point_prompt_generator(self):
         from micro_sam.util import get_cell_center_coordinates
 
         labels = self._get_test_data()
-        label_ids = np.unique(labels)[1:]
-        centers, boxes = get_cell_center_coordinates(labels)
+        label_ids = np.unique(labels)
+
+        mode = "v"  # this gets center points using vigra - regionprops throws error for asymmetrical blobs
+        centers, boxes = get_cell_center_coordinates(labels, mode=mode)
+        if mode == "v":
+            centers.pop(0)   # to avoid the background element
 
         test_point_pairs = [(1, 0), (1, 1), (2, 4), (3, 9)]
         for (n_pos, n_neg) in test_point_pairs:
             generator = PointPromptGenerator(n_pos, n_neg, dilation_strength=4)
-            for label_id in label_ids:
-                center, box = centers[label_id], boxes[label_id]
+            for label_id in label_ids[:-1]:  # we dodge the last element due to indexing issues for last val
+                center, box = centers[label_id], boxes[label_id]  # we start @id0 hence first mask prompts are accessed
                 coords, point_labels, _, _ = generator(labels, label_id, center, box)
                 coords_ = (np.array([int(coo[0]) for coo in coords]),
                            np.array([int(coo[1]) for coo in coords]))
-                mask = labels == label_id
+                mask = labels == label_id + 1  # here we start @id0 hence adding 1 (to ignore background)
```

Comment:
This is not a  good idea. We need to make sure that the `label_id` indexing matches the actual label ids.

Reply:
Ah, I made this change to adapt to the exact mask's configuration to access the `label_id` (because `label_ids` have 0 {background} included right now, and I access `label_id + 1`'s  mask because that's the first expected mask (cross-checked it w.r.t. the centers and bboxes too)).

Reply:
It's more readable to change the `label_ids` instead. (See my suggestion on top).